bin/drupalgap.js

```diff
@@ -6470,7 +6470,7 @@ function _drupalgap_page_title_pageshow_success(title) {
 function comment_menu() {
```

Comment:
@coolman7 `hook_menu()` is one of the spots where we should not use `t()`. I'll plan to merge this is for now, but we'll want to remove all `t()` calls from `hook_menu()`, and instead use `t()` during title render time.